cl_sii/rut/__init__.py

```diff
@@ -144,7 +145,10 @@ def __hash__(self) -> int:
     def clean_str(cls, value: str) -> str:
         # note: unfortunately `value.strip('.')` does not remove all the occurrences of '.' in
         #   'value' (only the leading and trailing ones).
-        return value.strip().lstrip('0').replace('.', '').upper()
+        clean_value = value.strip().replace('.', '').upper()
+        # Remove leading zeros excepts it is followed by a '-', so we can accept the RUT '0-0'
```

Comment:
Fix outdated comment.

```suggestion
        # Remove leading zeros except if zero is the only digit, so we can accept the RUT '0-0'.
```

Reply:
The commit message is also outdated (because the changed regular expression doesn't look for `-` anymore).

Reply:
Both fixed. Thanks